plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/maven2/Maven2GroupFacet.java

```diff
@@ -84,12 +88,28 @@ protected void doInit(final Configuration configuration) throws Exception {
     this.mavenFacet = facet(MavenFacet.class);
   }
 
+  @Override
+  protected void doConfigure(final Configuration configuration) throws Exception {
+    super.doConfigure(configuration);
+    if (cacheController == null) {
+      cacheController = new CacheController(1440, null);
+    }
+    cacheController.invalidateCache();
```

Comment:
In case of group, invalidateCache is called on configure to invalidate cached merged MDs due to member changes. In a moment we "align" proxy and maven2 group (with proxy to persist cacheToken in config, and with maven2 group to persist cacheToken and contentMaxAge in config) things will be more aligned.

Also, another difference: on proxy it is user explicit action to "expire" caches. On group, it is (currently) implication of group configuration (group member more precisely)  change.